wsl-pro-service/internal/system/system.go

```diff
@@ -70,9 +71,15 @@ func New(args ...Option) *System {
 		f(&opts)
 	}
 
-	return &System{
+	s := &System{
 		backend: opts.backend,
 	}
+
+	if err := s.EnsureValidLandscapeConfig(context.Background()); err != nil {
```

Comment:
So here we ensure the fixes are done at startup-time, right?

Reply:
Exactly! We initialize the "system" and so this at startup.